data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1366,23 +1411,26 @@ mission "Hai Reveal [B13] Freedom"
+				goto finished
 			label knowduty
 			`	They lift their head and smile. "A promise made is a promise kept. If your word is your bond then it would not do for me to stand in its way.`
+				goto finished
 			label finished
 			`	"The missing leader you seek is embedded within the Syndicate and is known by the name Terry Adrianopoulos."`
 			choice
-				`	(Be surprised.)`
+				`	"Terry did this!?"`
 					goto surprised
-				`	(You suspected.)`
+				`	"I suspected as much."`
 			`	Their words scratch an itch of suspicion that you didn't realize had been irritating you until now. They smirk knowingly. "I have merely confirmed that which you already guessed. Very well, I need say no more. This matter is yours to do with as you will." They leave you to ponder your course.`
 				goto done
 			label surprised
-			`	Your face must have revealed your response, for they smile at you with subtle mirth. "I see you know of this person, and did not suspect. I think it is to you and you alone I shall impart this knowledge. Do with it as you will." They leave you to ponder your course.`
+			`	They smile at you with subtle mirth at your surprise. "I see you know of this person, and did not suspect. I think it is to you and you alone I shall impart this knowledge. Do with it as you will." They leave you to ponder your course.`
 			label done
 			`	As you return to your ship, Teeneep interrupts you. "I need you to escort me one last time, Captain <last>. I must return to <planet> to resolve things now that the threat is ended. You will be well-paid, of course."`
-			`	Anticipating your next question, she says, "The ships of the Unfettered Hai will be escorted back with us since the Elenctics are convinced they are at no risk and still have a ship at their disposal. Some of those who come as crew on the Unfettered vessels have chosen to remain here and join the Elenctics. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. Danforth will come with us, but the other human vessels will make their way home themselves. Now we must rejoin our people on <planet> and set about resolving this matter for good, finally."`
+			`	Anticipating your next question, she says, "The ships of the Unfettered Hai will be escorted back with us, since the Elenctics are convinced they are at no risk and still have a ship at their disposal. Some of those who come as crew on the Unfettered vessels have chosen to remain here and join the Elenctics. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. Danforth will come with us, but the other human vessels will make their way home themselves. Now we must rejoin our people on <planet> and set about resolving this matter for good, finally."`
```

Comment:
```suggestion
			`	Anticipating your next question, she says, "The ships of the Unfettered Hai will be escorted back with us, since the Elenctics are convinced they are at no risk and still have a ship at their disposal. Some of those who came as crew on the Unfettered vessels have chosen to remain here and join the Elenctics. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. Danforth will come with us, but the other human vessels will make their way home themselves. Now we must rejoin our people on <planet> and set about resolving this matter for good, finally."`
```